internal/exec/validate_stacks.go

```diff
@@ -86,11 +89,29 @@ func ValidateStacks(cliConfig schema.CliConfiguration) error {
 			atmosManifestJsonSchemaFilePath = cliConfig.Schemas.Atmos.Manifest
 		} else if u.FileExists(atmosManifestJsonSchemaFileAbsPath) {
 			atmosManifestJsonSchemaFilePath = atmosManifestJsonSchemaFileAbsPath
+		} else if u.IsURL(cliConfig.Schemas.Atmos.Manifest) {
+			tempDir := os.TempDir()
+			fileName, err := u.GetFileNameFromURL(cliConfig.Schemas.Atmos.Manifest)
+			if err != nil {
+				return fmt.Errorf("failed to get the file name from the URL '%s': %w", cliConfig.Schemas.Atmos.Manifest, err)
+			}
+			atmosManifestJsonSchemaFilePath = path.Join(tempDir, fileName)
```

Comment:
Using os.CreateTemp() will result in the creation of multiple files because it generates a uniquely named file each time it is called
simply we overwrite the file each time